src/ServicePulse.Host/vue/tsconfig.node.json

```diff
@@ -8,7 +8,8 @@
     "composite": true,
     "noEmit": true,
     "module": "ESNext",
-    "moduleResolution": "Bundler",
-    "types": ["node"]
+    "moduleResolution": "Node",
```

Comment:
Same, @cquirosj can you explain why we need to change this at all?

Reply:
Had to pick one, Bundler was not a valid option

<img width="972" alt="image" src="https://github.com/Particular/ServicePulse/assets/545695/27a1b61c-2c50-42ec-8e3d-7e250f2c468c">

Reply:
I don't know what tool you are using, but bundler is valid, see https://www.typescriptlang.org/tsconfig#moduleResolution.
You may need to update your tooling.
Also if you look at the error message, it actually says it!
<img width="771" alt="image" src="https://github.com/Particular/ServicePulse/assets/122651/2da9acfa-5464-4cfa-8978-5a6105867108">